kalite/contentload/__init__.py

```diff
@@ -0,0 +1 @@
+# DO NOT PUT ANYTHING HERE
```

Comment:
Why are we not putting anything here? There's a dearth of explanations on this PR.